app/src/main/java/com/kickstarter/ui/viewholders/discoverydrawer/ParentFilterViewHolder.java

```diff
@@ -43,7 +50,7 @@ public void bindData(final @Nullable Object data) throws Exception {
   public void onBind() {
     final Context context = context();
 
-    filterTextView.setText(item.params().filterString(context));
+    filterTextView.setText(item.params().filterString(context, ksString, false, true));
```

Comment:
This is a very good suggestion. It did feel a little dated to be injecting `KSString` into all these files. Thanks!